Packs/MicrosoftExchangeOnline/ReleaseNotes/1_6_7.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Fixed an issue that caused duplicate fetching of read emails.
+- Resolved a problem where incidents were not fetching properly.
+- Fixed an issue where fetching incidents by modification time was unreliable, which could result in missed or duplicate incidents.
```

Comment:
Let's rephrase this RN. The second point is too abstract - what do you mean by not fetching properly?